src/apollo/queries.js

```diff
@@ -668,6 +668,30 @@ export const PAIR_DATA = (pairAddress, block) => {
   return gql(queryString)
 }
 
+export const MINING_POSITIONS = account => {
+  const queryString = `
+    query users {
+      user(id: "${account}") {
+        miningPosition {
+          id
+          user {
+            id
+          }
+          miningPool {
```

Comment:
Question about this subgraph schema, could the miningPool field be replaced with just a pointer to "pair"? or is there more data within that entity

Reply:
miningPool refers to the new staking contracts and includes information such as total staked which ws needed in order to make these calculations

Reply:
Ah yep, makes sense

Reply:
Ah yep makes sense